sample-operators/tomcat-operator/src/main/java/io/javaoperatorsdk/operator/sample/webapp/WebappReconciler.java

```diff
@@ -48,11 +52,20 @@ public void prepareEventSources(EventSourceRegistry<Webapp> eventSourceRegistry)
           // we need to find which WebApp this Tomcat custom resource is related to.
           // To find the related customResourceId of the WebApp resource we traverse the cache to
           // and identify it based on naming convention.
-          return eventSourceRegistry.getControllerResourceEventSource().getResourceCache()
-              .list(webApp -> webApp.getSpec().getTomcat().equals(t.getMetadata().getName()))
-              .map(ResourceID::fromResource)
+          var webAppInformer =
```

Comment:
Are you up to date with master, this should  not be changed probably.

Reply:
Is it okay to close and reopen this pull-request? The refactoring is a little different from what I originally intended, so I think it would be better to re-pullrequest it again.

Reply:
sure, np